docs/articles/documentation/test-api/assertions/assertion-api.md

```diff
@@ -361,6 +505,18 @@ Parameter              | Type                                              | Des
 await t.expect('foobar').match(/^f/, 'this assertion will be passed');
 ```
 
+```js
+import { ClientFunction } from 'testcafe';
+
+fixture `My fixture`;
+
+test('My test', async t => {
+    const getLocation = ClientFunction(() => document.location.href.toString());
+
+    await t.expect(getLocation()).match('/.com/');
```

Comment:
the `.` symbol should be escaped in a regexp. Otherwise it means 'any symbol':
`/\.com/`